libs/dh/dev-examples/feature-processes/src/components/details.component.ts

```diff
@@ -94,7 +94,7 @@ import { DhCalculationsDetailsGridAreasComponent } from './gridareas.component';
           @if (loading()) {
             {{ t('loading') }}
           } @else if (calculationType()) {
-            {{ 'calculationTypes.' + calculationType() | transloco }}
+            {{ 'shared.calculationTypes.' + calculationType() | transloco }}
```

Comment:
question: is there still a need for `devExamples.processes.calculationTypes` if these are looked up in `shared` now.